### ISSUE TITLE
Missing DOCS directory?

### ISSUE THREAD

**Issue by eli-oat** [Bushmills/jsforth#4]
[`jsforth80x25.html`](https://github.com/Bushmills/jsforth/blob/master/jsforth80x25.html) includes an iframe referencing [`DOCS/`](https://github.com/Bushmills/jsforth/blob/009f52580aaf0ca04d4a8970eb9ee88b992d4641/jsforth80x25.html#L30) directory that isn't included in the repo. Is that existent but missing from the repo?

**Comment by Bushmills** [comment]
Possible that location didn't survive migration to github, but more likely is that I excluded it from upload, because of the very unfinished nature of the documentary files in there. I just checked contents, and those files are for the most parts just stubs, about 100 words with just the stack effect, but no description, and the remaining about 250 words completely empty (but present, 0 byte sized files). Probably the contents of that directory were waiting for a better opportunity to see daylight than the very preliminary state they're currently in.
